manuscript/algorithm.tex

```diff
@@ -181,3 +190,68 @@ \subsubsection{\gls{MARKAL}}
 of all the coefficients for the \gls{LP} problem, and is
 solved. 
 
+
+\subsection{Stochastic-Optimizing (SO)}
+Stochastic optimization refers to optimization methods
+that incorporate randomness in either the objective function
+or the constraints \cite{lauren_stochastic_2015}. It aims to find
```

Comment:
You must update the bib file "
  Citation `bergamaschi_a_2011' on page 4 undefined on input line 227
  Citation `dorfman_linear_1958' on page 4 undefined on input line 169
  Citation `engel_can_1994' on page 4 undefined on input line 226
  Citation `krolzig_a_2002' on page 4 undefined on input line 227
  Citation `lauren_stochastic_2015' on page 4 undefined on input line 197
  Citation `melo_gaussian_2012' on page 5 undefined on input line 248"